controllers/noobaa-rules.yaml

```diff
@@ -1,56 +1,58 @@
+           max(NooBaa_num_buckets_claims{namespace="redhat-data-federation"}) without (instance)
          labels:
            origin: data-federation-service
-         record: noobaa_num_buckets_claims
+         record: namespace:noobaa_num_buckets_claims:max
        - expr: |
-           NooBaa_num_unhealthy_bucket_claims{namespace="redhat-data-federation"}
+           max(NooBaa_num_unhealthy_bucket_claims{namespace="redhat-data-federation"}) without (instance)
          labels:
            origin: data-federation-service
-         record: noobaa_num_unhealthy_bucket_claims
+         record: namespace:noobaa_num_unhealthy_bucket_claims:max
        - expr: |
-           NooBaa_num_unhealthy_namespace_buckets{namespace="redhat-data-federation"}
+           max(NooBaa_num_unhealthy_namespace_buckets{namespace="redhat-data-federation"}) without (instance)
          labels:
            origin: data-federation-service
-         record: job:noobaa_total_unhealthy_namespace_buckets
+         record: namespace:noobaa_total_unhealthy_namespace_buckets:max
        - expr: |
-           NooBaa_num_namespace_buckets{namespace="redhat-data-federation"}
+           max(NooBaa_num_namespace_buckets{namespace="redhat-data-federation"}) without (instance)
          labels:
            origin: data-federation-service
-         record: job:noobaa_namespace_bucket_count
+         record: namespace:noobaa_namespace_bucket_count:max
        - expr: |
-           NooBaa_accounts_num{namespace="redhat-data-federation"}
+           max(NooBaa_accounts_num{namespace="redhat-data-federation"}) without (instance)
          labels:
            origin: data-federation-service
-         record: noobaa_accounts_num
+         record: namespace:noobaa_accounts_num:max
        - expr: |
-           NooBaa_total_usage{namespace="redhat-data-federation"}
+           max(NooBaa_total_usage{namespace="redhat-data-federation"}) without (instance)
          labels:
            origin: data-federation-service
-         record: noobaa_total_usage
+         record: namespace:noobaa_total_usage:max
        - expr: |
-            NooBaa_num_unhealthy_namespace_resources{namespace="redhat-data-federation"}
+            max(NooBaa_num_unhealthy_namespace_resources{namespace="redhat-data-federation"}) without (instance)
          labels:
            origin: data-federation-service
-         record: noobaa_num_unhealthy_namespace_resources
+         record: namespace:noobaa_num_unhealthy_namespace_resources:max
        - expr: |
-           NooBaa_num_namespace_resources{namespace="redhat-data-federation"}
+           max(NooBaa_num_namespace_resources{namespace="redhat-data-federation"}) without (instance)
          labels:
            origin: data-federation-service
-         record: noobaa_num_namespace_resources
+         record: namespace:noobaa_num_namespace_resources:max
        - expr: |
            NooBaa_namespace_resource_status{namespace="redhat-data-federation"}
```

Comment:
please define how many series this expression could return. We also need a max() function wrapping it.

Reply:
Hello Simon. After our call today, @cloudbehl suggested we remove this metrics entirely, since we do not need the status for every resource at all times, only when there's something wrong, which is already covered by alerts.

Instead of this metric, we have bifurcated this into a count of healthy and unhealthy number of resources, so the cardinality for all the metrics now == 1.